test/integration_test/utils/storagecluster.go

```diff
@@ -168,6 +168,22 @@ func UpdateAndValidateAutopilot(cluster *corev1.StorageCluster, f func(*corev1.S
 	return latestLiveCluster
 }
 
+// UpdateAndValidateMonitoring update StorageCluster, validates Monitoring components only and return latest version of live StorageCluster
+func UpdateAndValidateMonitoring(cluster *corev1.StorageCluster, f func(*corev1.StorageCluster) *corev1.StorageCluster, pxSpecImages map[string]string, t *testing.T) *corev1.StorageCluster {
+	liveCluster, err := operator.Instance().GetStorageCluster(cluster.Name, cluster.Namespace)
+	require.NoError(t, err)
+
+	newCluster := f(liveCluster)
+
+	latestLiveCluster, err := UpdateStorageCluster(newCluster)
+	require.NoError(t, err)
+
+	err = testutil.ValidateMonitoring(pxSpecImages, latestLiveCluster, DefaultValidateAutopilotTimeout, DefaultValidateAutopilotRetryInterval)
```

Comment:
Can we rename these constants everywhere `DefaultValidateAutopilotTimeout`? Instead of calling Autopilot, may be just say `DefaultValidateComponentTimeout`?

Reply:
Done